collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: review
+title: Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results
+tags: machine-learning, neural and evolutionary computing
+author: Tarvainen Antti, Valpola Harri
+cite:
+    authors: "Tarvainen et Valpola"
```

Comment:
You can give the full name of the authors here, since they're not repeated anymore :wink: 
```suggestion
    authors: "Tarvainen Antti, Valpola Harri"
```